types/tests/package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why are `npm` files included in the PR?

Reply:
These tests are not suitable until we add chain ID and access list